components/pages/workPageLayout/helper.ts

```diff
@@ -101,3 +119,55 @@ export const getIconNameFromMaterialType = (materialType: GeneralMaterialTypeCod
     return "podcast"
   }
 }
+
+export const slideSelectOptionsFromMaterialTypes = (workMaterialTypes: GeneralMaterialType[]) => {
+  return workMaterialTypes.map(materialType => {
+    return {
+      code: materialType.code,
+      display: translateMaterialTypesStringForRender(
+        materialType.code as GeneralMaterialTypeCodeEnum
+      ),
+    }
+  }) as SlideSelectOption[]
+}
+
+export const sortSlideSelectOptions = (options: SlideSelectOption[]) => {
+  return options.sort((a, b) => {
+    // sort by the index of the GeneralMaterialTypeCodeEnum in the materialTypeSortPriority array
+    return (
+      goConfig("materialtypes.sortpriority").indexOf(a.code) -
+      goConfig("materialtypes.sortpriority").indexOf(b.code)
+    )
+  })
+}
+
+export const getManifestationMaterialTypeIcon = (manifestation: ManifestationWorkPageFragment) => {
+  const materialType = getManifestationMaterialType(manifestation)
+  return getIconNameFromMaterialType(materialType.code) || "book"
```

Comment:
Is it possible to use a constant, either from our code or from generated code instead of just the "random" string `"book"`?

Reply:
I can write a comment on why we return "book" 😉